scripts/update_apkid.sh

```diff
@@ -0,0 +1,28 @@
+script_path=$(dirname $0)
+if [ "$script_path" != "scripts" ] && [ "$script_path" != "./scripts" ]; then
+    echo 'Please run script from MobSF directory '
+    echo './scripts/update_apkid.sh '
+    exit  1
+fi
+current_dir=$(pwd)
+rules_dir=${current_dir}/venv/lib/python3.6/site-packages/apkid/rules/
```

Comment:
What if this directory is different for user?

Reply:
we need an absolute path in python script so I need to calculate the path

Reply:
use a better way to calculate path